### PR TITLE
front-end structure

### PR DESCRIPTION
**Pull request by RoderickVongsa**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,34 @@
...
+    <header><Header /></header>
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/catindex" element={<CatIndex />} />
+        <Route path="/catshow" element={<CatShow />} />
+        <Route path="/catnew" element={<CatNew />} />
+        <Route path="/catedit" element={<CatEdit />} />
+        <Route path="*" element={<NotFound />} />
+      </Routes>
+      <footer><Footer /></footer>
```
Comment (sjproctor) [inline src/App.js]: Your component calls for Footer and Header don't need to be wrapped in a tag. If you use the header and footer tags, they would live in the actual Footer and Header component. Both of those tags function like div tags. They are just container tags.

**Review by sjproctor — COMMENTED** [review]
Just remove the tags that are wrapping your Header and Footer component and you'll be good to go.

File: src/pages/NotFound.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    backgroundSize: 'cover',
+    backgroundColor: '#0000FF',
+    height: '100vh',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    textAlign: 'center',
+    color: 'white',
+    fontSize: '2rem',
+  };
```
Comment (elysemontano) [inline src/pages/NotFound.js]: Setting up styles on a component is definitely an approach that some developers take and am curious what brought you to going this approach versus using your CSS file and calling on classnames?

**Review by elysemontano — COMMENTED** [review]
Left a comment in your NotFound. Not a requested change, but would like to know your thought process on your approach before approving.

**Review by elysemontano — CHANGES_REQUESTED** [review]
One last thing and you will be good to go. Check indentation in your NotFound page. A helpful way to keep consistent with formatting is to select the entire file, right click, and select format.

**Comment by RoderickVongsa** [comment]
got it will make a change to it!

**Review by elysemontano — APPROVED** [review]

**Pull request merged**